pkg/columns/options.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package columns
+
+import "github.com/kinvolk/inspektor-gadget/pkg/columns/ellipsis"
+
+type Option func(*Options)
+
+type Options struct {
+	DefaultAlignment Alignment             // default text alignment to use
+	DefaultEllipsis  ellipsis.EllipsisType // default type of ellipsis to use for overflowing text
+	DefaultWidth     int                   // width to be used when no width is specified for a column
+	RequireColumnTag bool                  // if set to true, Columns will consider all struct members, regardless of the column tag being present
+}
+
+func GetDefault() *Options {
+	return &Options{
+		DefaultAlignment: AlignLeft,
+		DefaultEllipsis:  ellipsis.End,
```

Comment:
Do we want to add the ellipsis at the end by default?

cc @mauriciovasquezbernal @alban

Reply:
So, are we fine with the ellipsis at the end of the string by default? I think it's more useful to know that content has been truncated and losing one character than not knowing.

Reply:
I think ellipsis at the end is ok by default. But, wouldn't it affect the integration tests?

Reply:
Yeah, but truncating without the ellipsis would probably affect them as well.

Reply:
Yes, it does. It is the reason of https://github.com/kinvolk/inspektor-gadget/pull/934 CI failing:

```bash
NODE             NAMESPACE        POD                            CONTAINER                 PID     TPID    SIGNAL           RET    COMM            
fv-az28-19       test-sigsnoop-9… test-pod                       test-pod                  15269   15269   SIGURG           0      runc:[2:INIT]   
fv-az28-19       test-sigsnoop-9… test-pod                       test-pod                  15269   1       SIGURG           0      runc:[2:INIT]   
fv-az28-19       test-sigsnoop-9… test-pod                       test-pod                  15269   15283   SIGTERM          0      sh              
```

Reply:
I tried, and it is truncated. Therefore, I agree `ellipsis.End` is the correct way to go. 

At this point, given that the library is truncating by default, it should indeed autoscale by default too. By doing so, the columns will be truncated only if the terminal is not big enough.

Sorry @flyth for asking you to disable and now enable the autoscaling. I was missing the fact that the library is truncating by default 🙈.